client/src/featureform/format.py

```diff
@@ -16,4 +22,31 @@ def format_rows(format_obj, format_obj_2=None, format_obj_3=None, format_obj_4=N
 
 def format_pg(s=""):
     print(divider)
-    print(s)
\ No newline at end of file
+    print(s)
+
+
+class CliFormatter:
```

Comment:
@sdreyer, I think [Rich](https://github.com/Textualize/rich) is a great candidate for a full-featured CLI formatting library and I originally thought I'd include it in this PR as we discussed; however, given the scope of changes required to parse out formatting logic from data access logic, I think it would be best to remove it at this time.

I left this example here to demonstrate how we could wrap Rich in our own class to expose a generalized formatting API to the Python CLI.